chain-impl-mockchain/src/ledger/iter.rs

```diff
@@ -240,3 +240,110 @@ impl<'a> std::iter::FromIterator<Entry<'a>> for Result<Ledger, Error> {
         })
     }
 }
+
+#[cfg(test)]
+mod tests {
+
+    use crate::{
+        ledger::{Entry, Ledger},
+        testing::{
+            data::AddressData,
+            ledger::{self, ConfigBuilder},
+        },
+        transaction::Output,
+        value::Value,
+    };
+    use chain_addr::Discrimination;
+
+    #[test]
+    pub fn iterate() {
+        let faucet = AddressData::utxo(Discrimination::Test);
+
+        let message = ledger::create_initial_transaction(Output::from_address(
+            faucet.address.clone(),
+            Value(42000),
+        ));
+        let (_block0_hash, ledger) =
+            ledger::create_initial_fake_ledger(&[message], ConfigBuilder::new().build()).unwrap();
+
+        // FIXME: generate arbitrary ledger
+
+        for item in ledger.iter() {
+            match item {
+                Entry::Globals(globals) => {
+                    println!(
```

Comment:
Is it normal to use `println` in these tests?
This looks like it needs rustfmt, too.

Reply:
I don't really know, just move this test into new location.